src/store/getters.ts

```diff
@@ -208,6 +208,12 @@ export default {
     return (state.entityState === EntityState.HISTORICAL)
   },
 
+  /** Is True if the business is frozen */
+  isAdminFreeze (state: StateIF): boolean {
+    console.log('isAdminFreeze', state.adminFreeze)
+    return state.adminFreeze
```

Comment:
I don't see in this PR where this store value is declared, initialized, or set in any way. Are some files missing from this PR?

Reply:
It is being set here https://github.com/bcgov/business-filings-ui/blob/main/src/store/mutations.ts#L30

Reply:
OK, I found it. I did not realize this state variable existed previously and I expected it to be declared and initialized in this PR.

Declared: https://github.com/bcgov/business-filings-ui/blob/61870a728eef653a169c178170d0a7f3fb0c4d02/src/interfaces/state-interface.ts#L20

Initialized: https://github.com/bcgov/business-filings-ui/blob/61870a728eef653a169c178170d0a7f3fb0c4d02/src/store/state.ts#L14

I'm resolving this conversation.